modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -15,6 +15,11 @@ Before you install an {product-title} cluster on your vCenter that uses infrastr
 
 To install an {product-title} cluster in a vCenter, the installation program requires access to an account with privileges to read and create the required resources. Using an account that has administrative privileges is the simplest way to access all of the necessary permissions.
 
+[IMPORTANT]
+====
+A vCenter user configured in {product-title} must have `Read` permissions, at minimum, for vCenter resources such as `folder`, `host`, `datacenter`, `datastore folder`, and `datastore cluster`.
+====
```

Comment:
This misses an important detail that user must have read permission to vcenter itself.

Reply:
I clarified this:
> ...must have `Read` permissions, at minimum, **for vCenter and for** resources...

OK, @gnufied?

Reply:
A large percentage of bugs related to vSphere are because of wrong permissions and I do not think this note quite does the justice. I know in past we have said, we can't add any screenshots to openshift docs but I am thinking may we need seriously rethink how we document this. 

There are two things we are talking about here:

1. If cluster admin is creating a new role for vcenter user that will be used in Openshift then privielges we list here has to be associated with the role assigned to the user or group to which user belongs to.  These permissions are not enough. For example, I can't even login to vcenter using those permissions. :-)

2. There are second set of vcenter level permissions.  These permissions are different from privileges assigned to the role. We need to document both permissions very clearly.

Reply:
@gnufied I have revised this section based on our phone call and my understanding of what's currently missing. I realize that the updated instructions are somewhat vague because we cannot doc the vCenter UI workflow, but I hope it is more on the path towards a user's success. 
In a nutshell, I tried to convey these steps:

1. Create a vCenter role.
2. Apply necessary privileges (table).
3. Apply `read` permissions to  both entities and to vCenter. Check "Permissions" VMware GH doc for details. (_Need to verify that this link is okay in this doc, as we don't usually provide GitHub links._)
4. Verify that this works (user is on their own here).

WDYT?